modules/ob-compatibility-support-matrix.adoc

```diff
@@ -23,8 +23,8 @@ The link:https://access.redhat.com/support/offerings/techpreview[Technology Prev
 
 | Operator | Builds (Shipwright) | CLI | | |
 
-|1.1 | 0.13.0 (GA) | 0.13.0 (GA) | 1.13, 1.14, and 1.15        | 4.12, 4.13, 4.14, 4.15, and 4.16         | GA
-|1.0 | 0.12.0 (GA) | 0.12.0 (GA) | 1.12, 1.13, 1.14, and 1.15 | 4.12, 4.13, 4.14, 4.15, and 4.16         | GA
-|1.2 |  |  |        |          | GA
+|1.2 | 0.13.0 (GA) | 0.13.0 (GA) | 1.13, 1.14, 1.15, and 1.16 | 4.12, 4.13, 4.14, 4.15, 4.16, and 4.17 | GA
```

Comment:
Maintenance support for OpenShift Pipelines 1.13 ended on October 9 - we should not support it.

```suggestion
|1.2 | 0.13.0 (GA) | 0.13.0 (GA) | 1.14, 1.15, and 1.16 | 4.12, 4.13, 4.14, 4.15, 4.16, and 4.17 | GA
```